Fix: Promotions should involve PR and feature branch (#3516) (#3598)
DataBiosphere/azul: OPERATOR.rst

#. ::

>       git checkout develop

Reviewer: ```suggestion
      git checkout develop
      git pull -ff-only
```

      git checkout -b promotions/<yyyy-mm-dd>
      git push github promotions/<yyyy-mm-dd>

> #. File a PR on github from the new promotions branch which targets ``prod``.

Reviewer 2: ```suggestion
#. File a PR on Github from the new promotions branch which targets ``prod``.
```

Reviewer 3: Thanks for spotting that. I think this review probably shouldn't have been an approval though.

> #. File a PR on GitHub from the new promotions branch which targets ``prod``.

Reviewer: ```suggestion
#. File a PR on GitHub from the new promotions branch. The PR must target ``prod``.
```

#. ::

      git checkout develop
      git pull -ff-only
      git checkout -b promotions/<yyyy-mm-dd>
      git push github promotions/<yyyy-mm-dd>

#. File a PR on GitHub from the new promotions branch. The PR must target ``prod``.

#. Search for and follow any special ``[u]`` upgrading instructions that were added.

> #. When merging, follow the checklist and making sure to carry over any commit

Reviewer: ```suggestion
#. When merging, follow the checklist and make sure to carry over any commit
```
